benchmarks/fasta/php/bench.php

```diff
@@ -128,15 +129,16 @@ function makeRepeatFasta($s, $n) {
    'AGGCGGAGGTTGCAGTGAGCCGAGATCGCGCCACTGCACTCC' .
    'AGCCTGGGCGACAGAGCGAGACTCCGTCTCAAAAA';
 
+makeCumulative($iub);
+makeCumulative($homosapiens);
+
```

Comment:
Don't we give PHP an unfair advantage here? In the javascript version `makeCumulative()` is called once per call to `fastaRandom()`. Here it happens once per execution.